synapse/storage/schema/__init__.py

```diff
@@ -133,3 +133,17 @@
 This value is stored in the database, and checked on startup. If the value in the
 database is greater than SCHEMA_VERSION, then Synapse will refuse to start.
 """
+
+BACKGROUND_UPDATES_COMPAT_VERSION = (
+    # The replace_stream_ordering_column from 6001 must have run.
+    61
```

Comment:
This probably needs upgrade notes to let folks know to double check this has ran before upgrading?